pkgs/tools/admin/lxd/ui.nix

```diff
@@ -0,0 +1,42 @@
+{ mkYarnPackage
+, fetchFromGitHub
+, fetchYarnDeps
+, lib
+, ...
+}:
+mkYarnPackage rec {
+  pname = "lxd-ui";
+  version = "0.0.1";
+
+  src = fetchFromGitHub {
+    owner = "canonical";
+    repo = "lxd-ui";
+    rev = "c2e819a027d440cbb1cb9d450aad280dde68e231";
+    sha256 = "sha256-lEzGACSv6CpxnfkOcsdPrH6KRKDkoKv63m8Gsodk8uc=";
+  };
+
+  packageJSON = ./package.json;
```

Comment:
```suggestion
```

You don't need this. `fetchYarnDeps` is handling the node dependencies in a fixed output derivation below.

Reply:
Apparently I do! Without this the nixpkgs-review fails :(

Reply:
The default is already to read this file from `src`, which seems much better to me than copying it into nixpkgs. https://github.com/NixOS/nixpkgs/blob/5aac3cc9ef7304a5cdccba49fb174090ad2636a4/pkgs/development/tools/yarn2nix-moretea/yarn2nix/default.nix#L252

Reply:
It's not clear to me why you're getting a failure, but if I comment this line out I'm able to build the package successfully.

Reply:
Ok, I understand now. This file is necessary to avoid IFD (import from derivation).